src/blocks/recommendations/components/parsely-recommendations-title.js

```diff
@@ -0,0 +1,5 @@
+const ParselyRecommendationsTitle = ( { title } ) => (
+	<>{ title && <p className="parsely-recommendations-list-title">{ title }</p> }</>
```

Comment:
What do the opening and closing elements on this line do?

Reply:
They are required, else we get parsing errors.

Reply:
This is the short syntax for a React Fragment: https://reactjs.org/docs/fragments.html

They are technically required for all components, but sometimes if the component is simple enough, they can be omitted. This is not the case so they are needed. If we don't want them, we should rewrite this as:

``` ({title}) => title && <p>...</p>```

Reply:
I've pushed a simpler and easier to understand ternary.

Reply:
Marking this as resolved.